lib/ansible/executor/task_executor.py

```diff
@@ -605,6 +605,7 @@ def _get_connection(self, variables, templar):
                 try:
                     cmd = subprocess.Popen(['ssh','-o','ControlPersist'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
                     (out, err) = cmd.communicate()
+                    err = to_str(err)
```

Comment:
I understand, theses are really nice functions you got there, thanks a heap for your reply !